BrownianMotion/Continuity/IsKolmogorovProcess.lean

```diff
@@ -299,14 +297,30 @@ lemma lintegral_sup_rpow_edist_le_of_minimal_cover_two (hp : 1 ≤ p)
       ≤ 2 ^ d * M * c₁ * (2 * ε₀ * 2⁻¹ ^ m) ^ (q - d) / (2 ^ ((q - d) / p) - 1) ^ p := by
   sorry
 
+lemma ENNReal.rpow_finsetSum_le_finsetSum_rpow {p : ℝ} {ι : Type*} {I : Finset ι} {f : ι → ℝ≥0∞}
+    (hp : 0 < p) (hp1 : p ≤ 1) : (∑ i ∈ I, f i) ^ p ≤ ∑ i ∈ I, f i ^ p := by
```

Comment:
hp could be replaced by `0 ≤ p`, right?
Same question in `lintegral_sup_rpow_edist_le_sum_rpow_of_le_one`.